### ISSUE TITLE
How to Send a private message 

### ISSUE THREAD

**Issue by dbaltieri** [thanos1983/Python2-TCP-Chat-Server-Multiple-Clients#1]
Hello,

I'm trying to send a private message to an exclusive NickName, is there a tag or parameter to do it ?

**Comment by thanos1983** [comment]
Hello dbaltieri,

No there is no such option, the original idea was initially to be a common reference server / client scenario.

But you can add it if you want, it should not that difficult.

Let me know what you think.

BR / Thanos

**Comment by dbaltieri** [comment]
I agree that's functionality is not so hard to do. It will be a pleasure to contribute. Lets do it!

**Comment by thanos1983** [comment]
Let me know if you get stack somewhere so I could help if possible. :) (BR / Thanos)
